README.md

```diff
@@ -44,29 +72,50 @@ In this case the following environment variables are used:
 
-If this simple logger implementation isn't sufficient for your needs you can implement your own logger on top of `esp-println` - see https://docs.rs/log/0.4.17/log/#implementing-a-logger
+If this simple logger implementation isn't sufficient for your needs, you can implement your own logger on top of `esp-println`. See [Implementing a Logger section log documentaion]
 
-## defmt
+## `defmt`
 
 Using the `defmt-espflash` feature, esp-println will install a defmt global logger. The logger will
 output to the same data stream as `println!()`, and adds framing bytes so it can be used even with
 other, non-defmt output. Using the `defmt` feature automatically uses the Rzcobs encoding and does
 not allow changing the encoding.
 
-You can also use the `defmt-raw` feature that allows using any encoding provided by defmt, but
-does not add extra framing. Using this feature requires some care as the defmt output may become
-unrecoverably mangled when other data are printed.
+You can also use the `defmt-raw` feature that allows using any encoding provided by `defmt`, but
+does not add extra framing. Using this feature requires some care as the `defmt` output may become
+unrecoverably mangled when other data is printed.
+
+Follow the [`defmt` book's setup instructions] on how to
+set up `defmt`. Remember, the global logger is already installed for you by `esp-println`!
+
+# `esp-backtrace`
+
+`esp-println` is usually used alongside [`esp-backtrace`]. When using this
+two crates together, make sure to use the same communication methods for
+both dependencies. Table matching features:
+| `esp-println` | `esp-backtrace`     |
+| ------------- | ------------------- |
+| `uart`        | `print-uart`        |
+| `jtag_serial` | `print-jtag-serial` |
+| `rtt`         | `print-rtt`         |
 
-Follow the [defmt book's setup instructions](https://defmt.ferrous-systems.com/setup) on how to
-set up defmt. Remember, the global logger is already installed for you by esp-println!
+[`defmt`]: https://github.com/knurling-rs/defmt
+[`log` crate]: https://github.com/rust-lang/log
+[rzCOBS]: https://github.com/Dirbaio/rzcobs
+[`espflash`]: https://github.com/esp-rs/espflash
+[rtt]: https://wiki.segger.com/RTT
+[disable the default features]: https://doc.rust-lang.org/cargo/reference/features.html#the-default-feature
+[`esp-backtrace`]: https://github.com/esp-rs/esp-backtrace
+[Implementing a Logger section log documentaion]: https://docs.rs/log/0.4.17/log/#implementing-a-logger
+[`defmt` book's setup instructions]: https://defmt.ferrous-systems.com/setup
 
-### Troubleshooting linker errors
+# Troubleshooting linker errors
 
 If you experience linker errors, make sure you have *some* reference to `esp_println` in your code.
 If you don't use `esp_println` directly, you'll need to add e.g. `use esp_println as _;` to your
 import statements. This ensures that the global logger will not be removed by the compiler.
 
-## License
+# License
```

Comment:
Just curious about this change, it was consistent with all (most?) of the other repositories as it was, which we're not deviating from. Not a huge deal but, though.

Reply:
Changed it so it does look like it's a subsection of the previous h1 header. Happy to revert it if you think we should keep it as other repos.

Reply:
Yeah that's fine, as I said not a big deal. Just without context seemed like kind of a strange change 😁